src/main/java/org/glovo/service/OrderService.java

```diff
@@ -0,0 +1,74 @@
+package org.glovo.service;
+
+import org.glovo.model.Order;
+import org.glovo.model.Product;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+
+@Service
+public class OrderService {
+
+    private final List<Order> orders = new ArrayList<>();
+
+    public List<Order> getAllOrders() {
+        return new ArrayList<>(orders);
+    }
+
+    public Order getOrderById(String orderId) {
+        return orders.stream()
+                .filter(order -> order.getId().equals(orderId))
+                .findFirst()
+                .orElse(null);
+    }
+
+    public Order createOrder(Order order) {
```

Comment:
it is OrderService,  all methods here is related to order so no need to add word Order to the method name,